ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/TrieLogLayer.java

```diff
@@ -136,13 +136,13 @@ public static TrieLogLayer readFrom(final RLPInput input) {
       if (input.nextIsNull()) {
         input.skipNext();
       } else {
-        final Map<Hash, BonsaiValue<UInt256>> storageChanges = new TreeMap<>();
+        final Map<Hash, BonsaiValue<Bytes32>> storageChanges = new TreeMap<>();
         input.enterList();
         while (!input.isEndOfCurrentList()) {
           input.enterList();
           final Hash slotHash = Hash.wrap(input.readBytes32());
-          final UInt256 oldValue = nullOrValue(input, RLPInput::readUInt256Scalar);
-          final UInt256 newValue = nullOrValue(input, RLPInput::readUInt256Scalar);
+          final Bytes32 oldValue = nullOrValue(input, RLPInput::readUInt256Scalar);
+          final Bytes32 newValue = nullOrValue(input, RLPInput::readUInt256Scalar);
```

Comment:
is reading uint256 correct here, or should it read bytes32?

Reply:
`readBytes32Scalar` wasn't in the RLPInput interface, so I pulled it up.  It also exposed that the proofs were still on `UInt256`s